tsconfig.json

```diff
@@ -24,18 +24,21 @@
 	},
 	"include": [ "client", "server" ],
 	"exclude": [
-		"public",
-		"node_modules",
-		"**/node_modules/*",
-		"build",
-		"**/build/*",
 		"**/build-module/*",
 		"**/build-style/*",
+		"**/build/*",
 		"**/dist/*",
+		"**/node_modules/*",
```

Comment:
I had my doubts, but `globtester.com` seems to indicate that this does indeed match the project-level `node_modules` 👍